Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9.1
+FROM python:3.9.14-bullseye
```

Comment:
Can we upgrade this to 3.12?

Reply:
In my tests, "poetry install" fails with python 3.11 and 3.12 images (bullseye) :-(